pkg/render/kubecontrollers/kube-controllers.go

```diff
@@ -578,7 +623,10 @@ func (c *kubeControllersComponent) controllersDeployment() *appsv1.Deployment {
 			},
 		},
 	}
-	render.SetClusterCriticalPod(&(d.Spec.Template))
+
+	if !c.cfg.Tenant.MultiTenant() {
```

Comment:
Is this change needed? Why don't we want to set this in multi-tenant?

Reply:
I cannot create multiple instances of es-kubecontrollers with this setting.

Reply:
Okie doke.